src/test/java/io/fabric8/runsh/RunShLoaderTest.java

```diff
@@ -42,29 +36,19 @@ public void checkCopyScript() throws IOException {
         Path dest =  Files.createTempDirectory("run");
         RunShLoader.copyRunScript(dest.toFile());
 
-        Set<String> filesToCheck =
-            new HashSet<>(
-                Arrays.asList("run-java.sh", "container-limits",
-                              "debug-options", "java-default-options"));
-
-        for (String script : dest.toFile().list()) {
-            assertTrue(filesToCheck.remove(script));
-            String extracted = FileUtils.readFileToString(new File(dest.toFile(), script));
-            String stored = loadFromClassPath("/run-java-sh/fp-files/" + script);
-            assertEquals(stored, extracted);
-        }
-        assertEquals(0, filesToCheck.size());
+        String files[] = dest.toFile().list();
+        assertEquals(1, files.length);
+        String extracted = new String(Files.readAllBytes(new File(dest.toFile(), "run-java.sh").toPath()));
+        String stored = loadFromClassPath("/run-java-sh/fp-files/run-java.sh");
+        assertEquals(stored, extracted);
     }
 
     // ======================================================================================================
 
 
     private String loadFromClassPath(String location) {
-        try {
-            return IOUtils.toString(getClass().getResourceAsStream(location));
-        } catch (IOException e) {
-            throw new IllegalStateException("Internal: Cannot load " + location + ":" + e,e);
-        }
+        Scanner s = new Scanner(getClass().getResourceAsStream(location)).useDelimiter("\\A");
```

Comment:
Same here, closing the input stream would be better probably.

Reply:
Otherwise making the `loadFromClassPath` helper method package-private would be a way to avoid duplicating here.

Reply:
✅